app/public/model/data/FetchedDataManager.js

```diff
@@ -51,7 +51,7 @@ export default class FetchedDataManager {
         if (url === null) {
             url = this.router.getUrl();
             if (!url.searchParams.has(dataReqParams.rowsOnSite)) {
-                url = new URL(`${url.href}&${dataReqParams.rowsOnSite}=${this.model.userPreferences.rowsOnSite}`);
+                url = new URL(`${url.href}&${dataReqParams.rowsOnSite}=${this.model.parent.userPreferences.rowsOnSite}`);
```

Comment:
`userPreferences` have been moved up in the model hierarchy, hence the change here. The whole `FetchedDataManager` file is meant to be removed in the nearest future (once all the views use the new endpoints).